scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/Conversion.scala

```diff
@@ -0,0 +1,143 @@
+package scala.meta
+package internal.semantic
```

Comment:
I don't mind refactoring the structure of `scala/meta/internal/semantic`, but I find somewhat counterproductive to discuss incremental steps in the refactoring split over several pull requests. Let's have a dedicated chat about where we want to bring this module.